src/components/Title/Title.styles.js

```diff
@@ -0,0 +1,17 @@
+import styled from '@emotion/styled';
+
+export const StyledTitle = styled.h1`
```

Comment:
Sorry, I didn't see this before, but there should be a more semantic alternative to this, you should allow the proper use of the different title tags. I'll do a quick research and share with you a link on what I can find.